app/ide-desktop/lib/content/watch.ts

```diff
@@ -2,5 +2,11 @@ import bundler from './esbuild-config.js'
 // @ts-ignore
 import * as server from 'enso-gui-server'
 
-await bundler.watch()
-await server.start({ root: bundler.output_path, assets: bundler.output_path })
+const reloadListenerCode =
+    '(() => window.live_reload = window.live_reload ?? (new EventSource("/live-reload").onmessage = () => location.reload()))();'
```

Comment:
I'd either move this snippet to the server or extract at least the event source. Now this `/live-reload` is hardcoded in literals in two different packages.

Reply:
What is `window.live_reload`? Perhaps it should be mentioned in the documentation.

Reply:
Moved the code to separate file in-server and added a comment about the use of global.